src/monitor.rs

```diff
@@ -115,10 +115,12 @@ macro_rules! monitor_cmd_for {
 						return;
 					}
 
-					let addr = runtime::storage().election_provider_multi_phase().round();
-					let round = match api.storage().fetch(&addr, Some(hash)).await {
+					let round = match api.storage().fetch(&runtime::storage().election_provider_multi_phase().round(), Some(hash)).await {
 						Ok(Some(round)) => round,
-						Ok(None) => unreachable!("Round must always exist"),
+						Ok(None) => {
+							log::warn!(target: LOG_TARGET, "EPM::Round was not found at: {:?}", at.hash());
```

Comment:
I have seen this on my dev node when no signed phase occured yet.

Reply:
Yeah once the first round has not even started, this will be zero.